### DISCUSSION TITLE
Redirect the user to correct home page based on browser language

### DISCUSSION THREAD

**Discussion by nunomazer** [razonyang/hugo-theme-bootstrap#934] [Q&A] [answered]
Hello, I'm looking to implement automatic language-based redirection for my website's visitors. When someone accesses the homepage at https://mazer.dev/, I'd like them to be redirected to https://mazer.dev/pt if their browser language is set to 'pt' or 'pt-br'. Otherwise, they should be redirected to https://mazer.dev/en. I understand this can be done using JavaScript, but I was wondering if there's a built-in feature in the theme to achieve this functionality.

**Comment by razonyang** [comment, marked as answer]
Sounds good to me, available since `v1.0.0`.
